src/quantum/azext_quantum/vendored_sdks/azure_mgmt_quantum/operations/workspaces_operations.py

```diff
@@ -297,7 +297,7 @@ def _delete_initial(
         request = self._client.delete(url, query_parameters, header_parameters)
         response = self._client.send(request, stream=False, **operation_config)
 
-        if response.status_code not in [200, 201]:
```

Comment:
this files are autogenerated from the swagger so shouldn't be manually modified.

Reply:
Thanks @anpaz, I'll remove this change, and address this properly via a separate pull request: #11